src/OneWireHub.cpp

```diff
@@ -404,14 +404,15 @@ bool OneWireHub::recvAndProcessCmd(void)
 {
 
     // If the only slave is not multidrop compatible, pass all data handling to the slave
-    if(slave_count == 1){
+    if(slave_count == 1u){
 
         slave_selected = slave_list[getIndexOfNextSensorInList()];
-        // TODO: this might be expensive for weak uC and OW in Overdrive -> look into optimizations (i.e. preselect when only one device present?)
+        // TODO: this might be expensive for weak uC and OW in Overdrive and only one device emulated
+        //  -> look into optimizations (i.e. preselect when only one device present?)
 
-        if( slave_selected->MULTIDROP == false ){
+        if( slave_selected->skip_multidrop ){
             slave_selected->duty(this);
-            return (_error != Error::NO_ERROR);
+            return false;
```

Comment:
is that really helpful? My thinking is: 
- the 701c is doing these strange double resets + presence-detection (even with the original battery)
- during duty() onewire-hub detects an error like the second reset and raises Error::RESET_IN_PROGRESS
- your change would make the hub leave the hub.poll() routine and later reset error-state on re-entering .poll()
   - it gets really problematic if you do other stuff than just .poll()
   - in addition the time in reset but still in previous .poll() is not considered when timing the reset in current .poll()
   - chances for missed communication is higher imho
- the original solution acts on the error
  - it directly begins to time the ongoing reset (it even considers the lost time in .duty()) and will continue with presence detection (check .poll() and .checkReset() for reference)

Reply:
Understood and makes sense on the higher chances for missed communications.